container_pipeline/workers/linter.py

```diff
@@ -81,15 +81,37 @@ def lint(self):
             out, err = run_cmd_out_err(command)
             if err == "":
                 response = self.handle_lint_success(out)
+                self.logger.info(
+                    "Docker file linting successful going for job creation")
+                if not response["job_created"]:
+                    self.logger.warning(
+                        "Openshift project is not created putting it back to tube")
+                    if self.job.get("retry") is None:
+                        self.job["retry"] = 1
+                    else:
+                        self.job["retry"] = self.job.get("retry") + 1
+                    self.job["action"] = "start_linter"
+                else:
+                    self.logger.info("Deleting 'dockerfile' data from job")
+                    self.job["dockerfile"] = None
+                    self.job["retry"] = None
             else:
                 response = self.handle_lint_failure(err)
                 self.job["dockerfile"] = None
                 self.job["action"] = "notify_user"
-                self.queue.put(json.dumps(self.job), 'master_tube')
+
+            if self.job.get("retry") > self.MAX_RETRY:
+                self.job["dockerfile"] = None
+                self.job["action"] = "notify_user"
+                self.job["notify_email"] = "container-status-report@centos.org"
```

Comment:
Please don't change the notification email here because we get numerous SUCCESS/FAIL emails on this list in a day already. If we change the workflow and send OpenShift project creation failure mails to the list instead of sending to end user, we might end up missing such emails and until someone takes a close look or a user points out that hist/her image hasn't been built since some time, no one might be aware of larger underlying issue(s).

Reply:
done as discussed